src/App.js

```diff
@@ -1,16 +1,14 @@
 import React, { Component } from "react";
-import Routes from "./routes/index";
+// import Routes from "./routes/index";
 import "font-awesome/css/font-awesome.min.css";
 import "./App.css";
 import "./myOwn.css";
-// import { getItem } from "./services/Session";
-// import Navbar from "./containers/Dashboard/Navbar";
-// import Header from "./components/Header";
-// import { LOGIN } from "./constants/index";
+import QuizForm from "./containers/Dashboard/Forms/QuizForm";
 
 class App extends Component {
   render() {
-    return <Routes />;
+    return <QuizForm/>
```

Comment:
@Humnaa 
Create a separate route for forms e.g. `/quiz-form` where you show this specific form instead of replacing the actual routes in `App.js`.

Reply:
I have made new separate routes for quizForm `/quiz-form`